sphinx/source/docs/dev_guide/building.rst

```diff
@@ -75,42 +83,36 @@ the complete Bokeh library (see :ref:`devguide_python_setup`).
 
 However, if you are using BokehJS as a standalone JavaScript library, then
 the instructions below describe the process to iteratively build BokehJS with
-Grunt for development purposes.
-
-Building BokehJS with Grunt
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Gulp for development purposes.
 
-If you want to be able to run the commands below, you can either use
-``bokehjs/node_modules/.bin/grunt`` or globally install 
-the grunt command line interface::
+Building BokehJS with Gulp
+~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-    $ npm install -g grunt-cli
-
-There are three main Grunt commands for development (to be executed from the
-``bokehjs`` subdirectory):
+Below are the main Gulp commands for development (to be executed from
+the ``bokehjs`` subdirectory). To run these commands, you can either
+use ``bokehjs/node_modules/.bin/gulp`` or globally install the Gulp
+command line interface:
 
 .. code-block:: sh
 
-    $ grunt deploy
+    $ npm install -g gulp
 
-This will generate the compiled and optimized BokehJS libraries, and deploy
-them to the ``build`` subdirectory.
+To generate the compiled and optimized BokehJS libraries, and deploy
+them to the ``build`` subdirectory:
 
 .. code-block:: sh
 
-    $ grunt build
+    $ gulp build
 
-This will build the BokehJS sources without concatenating and optimizing into
-standalone libraries. At this point BokehJS can be be as an `AMD module`_.
+To direct Gulp to automatically watch the source tree for changes and
+trigger a recompile if any source file changes. This is especially
+useful together with the ``--splitjs`` mode of the Bokeh server to
```

Comment:
This is no longer true is it? Does `--splitjs` still work? I thought the browserify change broke it.